lib/ansible/modules/crypto/openssl_cert.py

```diff
@@ -0,0 +1,616 @@
+        description:
+            - Key/value pairs that must be present in the subject name field of the certificate
+    has_expired:
+        required: false
+        choices: [ True, False ]
+        description:
+            - Checks if the certificate is expired/not expired at the time the module is executed.
+    version:
+        required: false
+        description:
+            - Version of the certificate. Nowadays it should almost always be 3.
+    valid_at:
+        required: false
+        description:
+            - The certificate must be valid at this point in time.
+              The value must be ISO-8601 format, a UTC timezone must be specified and sub-second precision is not allowed, e.g. `2012-12-21T21:12:23Z`
+    invalid_at:
+        required: false
+        description:
+            - The certificate must be invalid at this point in time.
+              The value must be ISO-8601 format, a UTC timezone must be specified and sub-second precision is not allowed, e.g. `2012-12-21T21:12:23Z`
+    starts_at:
+        required: false
+        description:
+            - The certificate must start to become valid at this point in time.
+              The value must be ISO-8601 format, a UTC timezone must be specified and sub-second precision is not allowed, e.g. `2012-12-21T21:12:23Z`
+    expires_at:
+        required: false
+        description:
+            - The certificate must expire at this point in time.
+              The value must be ISO-8601 format, a UTC timezone must be specified and sub-second precision is not allowed, e.g. `2012-12-21T21:12:23Z`
+
+    warn:
+        required: false
+        default: True
+        choices: [ True, False ]
+        description:
+            - Return warnings if some checks couldn't be performed, usually due to old library versions.
+
+    selfsigned_digest:
+        required: false
+        default: "sha256"
+        description:
+            - Digest algorithm to be used when self-signing the certificate
+    selfsigned_notBefore:
+        required: false
+        default: 0
+        description:
+            - Number of seconds from now which the self-signed certificate will be valid from.
+    selfsigned_notAfter:
+        required: false
+        default: 315360000 (3650 days)
+        description:
+            - Number of seconds from now which the self-signed certificate will be expired.
+
+    letsencrypt_accountkey:
+        required: false
+        description:
+            - Path to the accountkey for the 'letsencrypt' provider
+    letsencrypt_challenge_path:
+        required: false
+        description:
+            - Path to the ACME challenge directory that is served on http://<HOST>:80/.well-known/acme-challenge/
+
+notes:
+    - Features/checks that will be skipped with a warning if the version of pyOpenSSL is below a certain version:
+      Check if public key of certificate and public key of private key are equal: requires pyOpenSSL >= 0.16.1
```

Comment:
The `: ` is confusing the yaml parser, you can remove it or quote the whole line.